### PR TITLE
completed ticket 7

### PR DESCRIPTION
**Pull request by manheicatherine**

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -4,7 +4,10 @@ export default function Comments({ comment }) {
...
     .toISOString()
     .replace("T", " ")
-    .replace("Z", "");
+    .replace("Z", "")
+    .split("")
+    .slice(0, -4)
+    .join("");
 
   return (
     <section>
```
Comment (elijaharley) [inline src/components/Comments.jsx:13]: Nice handling of timestamp formatting.

File: src/components/VotesArticle.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      <button
+        disabled={updatevotes === -1}
+        onClick={() => handleDisike()}
+        className="votesbutton"
+      >
+        -
+      </button>
+    </section>
+  );
+}
```
Comment (elijaharley) [inline src/components/VotesArticle.jsx:37]: Where you are handling the error here by console.logging it you can have another piece of state for error which renders conditionally whether the patch request is successful or not.

**Review by elijaharley — APPROVED** [review]
There is some advice inline about optimistic rendering, please use NCHelp if you are unsure of this - you seem to be very close.

File: src/components/VotesArticle.jsx (hunk 2 above)
Comment (manheicatherine) [reply src/components/VotesArticle.jsx:37]: Thank you Eli!

**Pull request merged**